lightbeam/__main__.py

```diff
@@ -13,6 +13,19 @@ def emit(self, record):
 
 DEFAULT_CONFIG_FILES = ['lightbeam.yaml', 'lightbeam.yml']
 
+# use a dictionary here so that command strings can be accessed with a lookup.
+#   This helps enforce usage of this structure
+ALL_COMMANDS = {
+   "validate": "validate",
+   "send": "send",
+   "validate+send": "validate+send",
+   "delete": "delete",
+   "truncate": "truncate",
+   "count": "count",
+   "fetch": "fetch",
+}
```

Comment:
Why use a dict here instead of a list? Curious for your rationale.

I honestly don't know if there's a Python convention either way.

Reply:
It's definitely clunky to read and to deal with the list as a whole. The main reason I did this was futureproofing: to make it clear to anyone implementing command-handling logic that all commands come from this list. If a future developer adds a new command, they'll know to add it to the global list instead of just adding another `if args.command == 'new_command` line.